docs/integrations/mirroring.md

```diff
@@ -234,6 +234,13 @@ def get_mapping_fields_command():
 
 ```
 
+## Classification and Mapping with Mirroring
+When incoming mirroring happens, the incident goes through both the classifier and the mapper. The classifier sets a specific value for the incident type, and the mapper updates the incident with this value, ensuring that the incident type matches what the classifier determined.
```

Comment:
When incoming mirroring occurs, the incident uses both the classifier and the mapper. The classifier sets a specific value for the incident type, and the mapper updates the incident with this value, ensuring that the incident type matches what the classifier determined.